PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackHist.cxx

```diff
@@ -52,8 +54,10 @@ AliFemtoDreamTrackHist::AliFemtoDreamTrackHist()
     fITShrdClsPileUp[i] = nullptr;
   }
 }
-AliFemtoDreamTrackHist::AliFemtoDreamTrackHist(bool DCADist, bool CombSig, bool TOFM)
-    : fMinimalBooking(false),
+AliFemtoDreamTrackHist::AliFemtoDreamTrackHist(bool DCADist, bool CombSig, bool TOFM, float fpTmin, float fpTmax)
+    : fpTmin(0.5),
+      fpTmax(4.05),
+      fMinimalBooking(false),
```

Comment:
I am a bit surprised that this actually works :P You use the same identifier fpTmin for a member and a argument. What you should do is change the name of the argument to pTmin (pTmax) and then assign that value to the member (e.g. fpTmin(pTmin), fpTmax(pTmax))